src/main/resources/com/cloudbees/jenkins/plugins/amazonecs/ECSCloud/config.jelly

```diff
@@ -58,6 +58,9 @@
     <f:entry field="maxMemoryReservation" title="${%Maximum declarative memoryReservation}" description="Maximum memoryReservation that can be configured in declarative agent. 0 means unlimited.">
       <f:textbox />
     </f:entry>
+    <f:entry field="maxNodes" title="${%Maximum Nodes}" description="Maximum number of nodes provisioned by ECS. 0 means unlimited.">
```

Comment:
`Node` isn't a term used elsewhere in the agent template configuration. I would change the description to "Maximum number of agents that can be provisioned from this template. 0 means unlimited."